src/test/java/pages/HomePage.java

```diff
@@ -3,18 +3,17 @@
 import org.openqa.selenium.By;
 import org.openqa.selenium.Keys;
 import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
 
 public class HomePage extends BasePage {
-   /* public HomePage(WebDriver givenDriver) {
+    public HomePage(WebDriver givenDriver) {
         super(givenDriver);
     }
     By userAvatarIcon = By.cssSelector("img.avatar");
     public WebElement getUserAvatar() {
         return findElement(userAvatarIcon);
-    }*/
-    public HomePage(WebDriver givenDrive){
-        super(givenDrive);
     }
+
     By firstPlaylist =By .cssSelector("* .playlist:nth-child(3)");

```

Comment:
`@FindBy(css = "* .playlist:nth-child(3)");`
 `private WebElement firstPlaylist;`

Please use the ` FindBy ` class for all the web element that has been used for your homework 23.